tests/migration/online_migration/yast2_migration.pm

```diff
@@ -209,10 +209,15 @@ sub run {
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
     send_key "alt-n";
-    # in leap to sle migration, we need click trust for package-hub's import-untrusted-gpg-key
-    if (is_leap_migration) {
+    # migration via smt will install packagehub and NVIDIA compute, we need click trust
+    # gpg keys; Same with leap to sle migration, need to trust packagehub gpg key.
+    if ((get_var('SMT_URL') =~ /smt/) || (is_leap_migration)) {
         assert_screen 'import-untrusted-gpg-key', 60;
         send_key 'alt-t';
+        if ((check_var('ARCH', 'x86_64')) && (!(is_leap_migration)) || (check_var('ARCH', 'aarch64'))) {
+            assert_screen 'import-untrusted-gpg-key-nvidia', 300;
```

Comment:
Do we need a new tag for nvidia? If no special reason why not just 'import-untrusted-gpg-key'?

Reply:
And do we need wait 300s for this screen?

Reply:
I have tired with the same tag as you say here, and I put two needles under the tag 'import-untrusted-gpg-key'. But when the workers' performance is not well, it caught the packagehub key needle twice though it had send_key 'alt-t'. If the worker's performance is not well, it cost time to continue after send key.

Reply:
Use wait_screen_change  {  assert_screen 'import-untrusted-gpg-key', 300; } You can try this instead of a new tag.

Reply:
> And do we need wait 300s for this screen?

Yes, I changed the time several times, when adding all modules here, it cost long time to search the repos and services.

Reply:
> Use wait_screen_change { assert_screen 'import-untrusted-gpg-key', 300; } You can try this instead of a new tag.

Thanks for your kindly suggestion, and I'll use it when I meet similar issues next time. For this ticket, I think it also solve the problem with a new tag.